sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/SortAggregateExec.scala

```diff
@@ -36,11 +40,12 @@ case class SortAggregateExec(
     initialInputBufferOffset: Int,
     resultExpressions: Seq[NamedExpression],
     child: SparkPlan)
-  extends BaseAggregateExec
+  extends AggregateCodegenSupport
   with AliasAwareOutputOrdering {
 
   override lazy val metrics = Map(
-    "numOutputRows" -> SQLMetrics.createMetric(sparkContext, "number of output rows"))
+    "numOutputRows" -> SQLMetrics.createMetric(sparkContext, "number of output rows"),
+    "aggTime" -> SQLMetrics.createTimingMetric(sparkContext, "time in aggregation build"))
```

Comment:
We need to collect `aggTime` metrics here as well, because (1).hash aggregate's `doProduceWithoutKeys()` already generates code for it, and (2).it's good to have anyway.